src/Sulu/Bundle/SnippetBundle/Admin/SnippetAdmin.php

```diff
@@ -103,11 +103,13 @@ function(Localization $localization) {
                 ->addOption('locales', $snippetLocales)
                 ->addAttributeDefault('locale', $snippetLocales[0]),
             (new Route('sulu_snippet.form', '/snippets/:locale/:id', 'sulu_admin.resource_tabs'))
-                ->addOption('resourceKey', 'snippets'),
+                ->addOption('resourceKey', 'snippets')
+                ->addOption('locales', $snippetLocales)
+                ->addAttributeDefault('locale', $snippetLocales[0]),
             (new Route('sulu_snippet.form.detail', '/details', 'sulu_admin.form'))
                 ->addOption('tabTitle', 'sulu_snippet.details')
                 ->addOption('backRoute', 'sulu_snippet.list')
-                ->addOption('locales', $snippetLocales)
+                ->addOption('locales', $snippetLocales) // TODO should be removed?
```

Comment:
@chirimoya What about this? It feels a little bit weird in here, because the parent route also already gets the locales... Should it be passed in a different way, like the `ResourceStore`?

Reply:
The merging is implemented in #3640. I will rebase this PR after #3640 has been merged.